lib/endpoint.js

```diff
@@ -299,6 +299,7 @@ class Endpoint extends Emitter {
       'file param is required and must be a string, array or PlaybackOptions object');
 
     let timeoutSecs = -1;
+    const playback_terminators = file.playback_terminators ? file.playback_terminators : 'none';
```

Comment:
so this eliminates the ability to call ep.set('playback_terminators') explicitly and then do ep.play.  So we are removing a feature here?